apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -304,12 +322,14 @@ private class HistoryEntry
                     }
                 }
             }
+            LogPerformanceDataAfterTest();
         }
 
         [Fact]
         public async Task ReadExactSingle()
         {
             await WriteSampleRowsAsync();
+            LogPerformanceDataBeforeTest();
             await Task.Delay(6);
```

Comment:
true re: logging.  but I'll just need to account for built in delays in the tests because I am moving the perf logging to be in attributes, so will also include the sample writing.
I'm not looking for regressions on the order of 6ms anyway.  more like ~2 seconds, which would indicate a shared session wasn't used (and a new one was created).